content/en/service_level_objective.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Service Level Objective
+status: Feedback appreciapted
+category: Property
+---
+
+Service level objective (SLO) is a target value or range of values for a service level that is measured by an [SLI](/service_level_indicator/). SLO is the objective that must be achieved and it reflects a smooth and successful user journey for the User of a service. 
+
+SLO is the key element for [Site Reliability Engineers](https://glossary.cncf.io/site_reliability_engineering/), [DevOps](https://glossary.cncf.io/devops/) and Business Teams to measure the quality of the user journey. SLO and [Error-budget](https://glossary.cncf.io/error_budget/) can be used to balance the delivery of new features to production. SLOs are targets set by engineering teams from the end-user perspective and not the operational or engineering team point of view.
```

Comment:
```suggestion
SLO is the key element for [Site Reliability Engineers](/site_reliability_engineering/), [DevOps](/devops/) and Business Teams to measure the quality of the user journey. SLO and Error-budget can be used to balance the delivery of new features to production. SLOs are targets set by engineering teams from the end-user perspective and not the operational or engineering team point of view.
```

- Updated to relative URIs
- CNCF Glossary does not have the term `Error-budget` yet. 😊